ohw22/seattle/index.md

```diff
@@ -0,0 +1,58 @@
+# US Northwest
+
+We will have two sub-programs in the US Northwest satellite, including a regular 5-day OHW satellite and a 3-week undergraduate summer program. 
+
+These two sub-programs are coordinated in their planning and coincides in time. Participatns are encouraged to interact fully across the groups and with other satellites.
+
+
+
+## Location
+​
+University of Washington, Seattle, WA
+​
+
+## Format
+
+In-person. We have space reserved to watch OHW tutorials together and work on projects. We also plan to have social gatherings outside of the formal meeting space to explore and enjoy the region together.
+
+
+## Regular OHW satellite (August 15-19, 2022)
+
+We welcome participants from all career stages and encourage applications from graduate students, postdocs and early career researchers. Advanced undergrad students with background in oceanography and data science are also welcome. Below is a summary of activities:
+
+1. We will watch all the tutorials together
+2. Participants can asynchronously work on projects with participants of other OHW events (either the global virtual event or other satellites). Or participants can work on projects synchronously with other participants in the two sub-programs in Seattle.
+3. We encourage students to communicate with the global virtual event and other satellite events to augment their OHW experience.
+
+
+**Logistics**
+
+We are able to provide lodging for a small number of participants who need to travel within the US Northwest region to the event. If you need support to travel, please let us know the reasons and provide an estimate of the cost in your application form. This request is _not_ part of the applicant evaluation process.
+
+**Contact**
+
+For questions specific to the regular OHW satellite, please email Dr. Wu-Jung Lee (leewj@uw.edu).
+
+
+## Undergraduate summer program (August 8-26, 2022)
+
+This "Data Science in Oceanography" undergraduate program will provide opportunities for undergraduate students in data-driven research in oceanography. Participating students will interact closely with faculty and graduate student mentors to develop and advance individual research projects. Students will fully participate in the OHW Northwest satellite event as part of this longer undergraduate program.
+
+See [details in the program flyer](./undergrad_flyer.pdf)!
```

Comment:
Somehow this link doesn't work, but I also don't know much about jekyll. Help!

Reply:
I'll get to it right now. BTW, we're using Sphinx now, not Jekyll.

Reply:
Oh funny, somehow i saw that jekyll wording somewhere. If you can push a fix here directly that'll be great. Thanks!

Reply:
There are some jekyll words and warnings from the afterlife still floating around. Ignore them :wink:

Reply:
Hmm. Sphinx refuses to build a link when the reference is not a full url (http ...). I suspect Sphinx doesn't like non-docs files mixed in with the docs files. I tested it with a regular png, with the same result.

My quick solution will be to move the pdf to the assets folder. Working on it